ssds/blobstore/__init__.py

```diff
@@ -13,10 +15,13 @@
 
 
 class BlobStore:
-    schema: typing.Optional[str] = None
+    schema: Optional[str] = None
 
     def upload_object(self, filepath: str, bucket: str, key: str):
         raise NotImplementedError()
 
     def list(self, bucket_name: str, prefix=""):
         raise NotImplementedError()
+
+    def get(self, bucket_name: str, key: str) -> bytes:
```

Comment:
I know in another PR, a method here returned `str`.  Just a suggestion, but would it make things more intuitive later to return `str` from all methods added here?

Reply:
Not sure I follow? This method is meant to returns bytes.

Reply:
I know, but I was suggesting the bytes be decoded into `str`.  Mostly just because I like to either do things entirely in bytes or entirely in str so that I can have consistent expectations when I run things in one codebase.  Part of this is trauma from all of the py2 -> py3 moves we did.  Not necessary though obviously.

Reply:
I see.

Encoding into str would make the data more difficult to use since it couldn't be straightforwardly written to a file without decoding back into bytes.

Some of the other methods in this class return objects that are best represented as strings, such as checksum hashes and blobstore keys.